deployment/roles/common/tasks/repo.yml

```diff
@@ -1,25 +1,23 @@
 ---
-- name: Create repo directory
-  file: 
-    path: "{{ bridge_path }}"
-    state: directory
-
 - name: Register files for copying
   shell: |
-    git ls-tree -r HEAD --name-only
+    git ls-tree -r HEAD --name-only | sed '/^contracts$/d'
     cd contracts; git ls-tree -r HEAD --name-only | sed -e 's/^/contracts\//'
-  register: rsync_files
+  register: registered_files
   delegate_to: 127.0.0.1
   become: false
   args:
-    chdir: ".."
+    chdir: "{{ lookup('env', 'PWD') }}/.."
```

Comment:
This does not change the behaviour of the playbook (since `pwd/..` is the same as `..`), but for some reason Molecule messes up paths so that this is required to make this tasks work in Molecule tests.